multicluster/test/yamls/test-acnp-copy-span-ns-isolation.yml

```diff
@@ -0,0 +1,24 @@
+apiVersion: multicluster.crd.antrea.io/v1alpha1
+kind: ResourceExport
+metadata:
+  name: strict-namespace-isolation-for-test-clusterset
+  namespace: antrea-mcs-ns
+spec:
+  kind: AntreaClusterNetworkPolicy
+  name: strict-namespace-isolation
+  clusternetworkpolicy:
+    priority: 1
+    tier: securityops
+    appliedTo:
+      - namespaceSelector: {}
+    ingress:
+      - action: Pass
+        from:
+          - namespaces:
+              match: Self
+          - podSelector:
+              matchLabels:
+                k8s-app: kube-dns
```

Comment:
It does not need to allow kube-dns for the test to pass. I am merely trying to align the yaml with https://github.com/antrea-io/antrea/blob/main/docs/antrea-network-policy.md#acnp-for-strict-namespace-isolation

Reply:
But I didn't see this in that doc. And the rule should only make sense if it's egress rule, no reason to allow kube-dns accessing all Pods.

Reply:
>  But I didn't see this in that doc.

Sorry I was meant to say in here https://github.com/antrea-io/antrea/pull/3512. I've created a PR to update the sample yamls of Antrea-native policies.

> And the rule should only make sense if it's egress rule, no reason to allow kube-dns accessing all Pods.

I was thinking in the lines of the DNS return packets need to be allowed. But I guess your point is that does not warrant an ingress rule since there's going to be some bypass flow. In that case let me just remove this section from ingress then.

Reply:
Yes, the policy is stateful, DNS return packets will be allowed if the requests are allowed regardless of the existence of this rule. The rule will only apply to packets in request direction.